src/main/java/frc/robot/RobotContainer.java

```diff
@@ -68,6 +69,18 @@ private void configureButtonBindings() {
     // Zeroes the heading when the start button is pressed
     new JoystickButton(m_driveController, Button.kStart.value).whenPressed(() -> m_swerveDriveSubsystem.zeroHeading(),
         m_swerveDriveSubsystem);
+
+    // Aims at target while the A button is held.
+    new JoystickButton(m_driveController, Button.kA.value)
+        .whenHeld(new LimelightAimingCommand(m_swerveDriveSubsystem, 0));
+
+    // Aims at blue balls while the B button is held.
+    new JoystickButton(m_driveController, Button.kB.value)
+        .whenHeld(new LimelightAimingCommand(m_swerveDriveSubsystem, 1));
+
+    // Aims at red balls while the X button is held.
+    new JoystickButton(m_driveController, Button.kX.value)
+        .whenHeld(new LimelightAimingCommand(m_swerveDriveSubsystem, 2));
```

Comment:
I think we can get the [alliance color from DriverStation](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/DriverStation.html), so instead of having two separate buttons for each color, we should just have one button to aim at the balls. The pipeline can be determined using the alliance color.